### ISSUE TITLE
Feature: webhook config obj for CLI

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#963]
There should be a better way to register webhooks and change webhooks without restarting sessions

- [x] Right now, you can only set one single webhook for all events. This differs from modern web standards for webhooks. You should be able to set as many webhooks as you'd like each with a selectable range of possible events.
- [x] Allow webhook registrations to consume basic authentication (maybe there is a username and password for post requests at the webhook address)
- [x] CLI: consume a file (for example `webhooks.json`) that sets up webhooks
- [x] CLI: If only `-w` is provided, then register all events there.
- [x] `removeWebhook` by webhook ID
- [x] Implement `updateWebhook`

**Comment by joedutra** [comment]
<img src="https://media.giphy.com/media/sNZddQMtrqpS0HEEMC/giphy.gif"></img>
Man, you know it sucks, how we copy your repo, if we have functions that you don’t even have, all for free, we perform almost every day, I came from Sulla and you too, but if you look at following the week, we redid it all lib again, now you will no longer be able to copy us, you copy and keep saying that we copy, nor compete with you ours and for free and your payment

**Comment by joedutra** [comment]
I come to apologize and say that we have nothing against this, only that they accuse us of copying and we never copied anything, if someone copied it was when someone changed our lib, I apologize and the intention was not to be slandered anymore of copying something we worked on every day. just like you, we will not do that anymore.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.9.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.8.0...3.9.0) (2021-03-08)

- 💚 rollback local machine npm version to `6.14.11` to match CI [`b1fb6f2`](https://github.com/open-wa/wa-automate-nodejs/commit/b1fb6f222a35e810eed303b46d10609b6df5c30b)
- ➕ added `uuid` dep #1449 [`a074e6b`](https://github.com/open-wa/wa-automate-nodejs/commit/a074e6b9db94c823f3559606e4f8870c86927577)
- update package lock [`f01fa6c`](https://github.com/open-wa/wa-automate-nodejs/commit/f01fa6c6cfac578c5746476d65f39d06c5b0b4d0)
- Release 3.9.0 [`61cd2b8`](https://github.com/open-wa/wa-automate-nodejs/commit/61cd2b8f1bc86c6735f4c7462324a1d1ee5334cf)
- 💥BREAKING💥 ✨ Improved webhook strategy✨  see: #963 [`796f3cb`](https://github.com/open-wa/wa-automate-nodejs/commit/796f3cb5d33b8db932530451024303d0e5e6abcf)
- ✨[CLI] `--webhook``-w` can now refer to a `json` file with array of webhook objects along with a single URL. default: `webhooks.json` #963 [`6918e4a`](https://github.com/open-wa/wa-automate-nodejs/commit/6918e4a4e6a941ca396a5a8d5779572c399e7bef)
- ✨ all session events now have an `id` #1449 [`5f6ef76`](https://github.com/open-wa/wa-automate-nodejs/commit/5f6ef7634c99a1cc6f1457f8b98d5fd91c4668b2)
- 💥 BREAKING💥 ✨[CLI]: `-e``--e` now requires an explicit webhook URL #963 [`11ad384`](https://github.com/open-wa/wa-automate-nodejs/commit/11ad3849bdcd6e2aa8df0030198763cd2f533290)
- 💚  ci fix [`e1a1a44`](https://github.com/open-wa/wa-automate-nodejs/commit/e1a1a443a0ae0c7b5f8ee613b6a581cd0978e1f0)
- Update comment-run.yml [`fa56136`](https://github.com/open-wa/wa-automate-nodejs/commit/fa56136fd6598688126951a4341de4c77317b852)
- 🥚  typo fix [`f3ca324`](https://github.com/open-wa/wa-automate-nodejs/commit/f3ca324eb94f35b7cd33289aef3f569c2b7e370f)

**Comment by smashah** [comment]
https://docs.openwa.dev/classes/client.html#updatewebhook
https://docs.openwa.dev/classes/client.html#registerwebhook
https://docs.openwa.dev/classes/client.html#removewebhook
https://docs.openwa.dev/classes/client.html#listwebhooks
